package.json

```diff
@@ -65,7 +66,7 @@
     "ethereum-waffle": "^3.1.1",
     "ethers": "5.6.2",
     "googleapis": "107",
-    "hardhat": "2.8.0",
+    "hardhat": "2.12.0",
```

Comment:
Does tenderly export work in that hardhat version?

Reply:
No. But, let's create a ticket for next week to address

Reply:
Georgi has pushed ahead with new version so will mark as resolved now